index.html

```diff
@@ -0,0 +1,194 @@
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project5 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project6 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project7 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="about-me-section">
+            <div class="grid-wrapper">
+                <div class="aboutme">
```

Comment:
- [ ] [OPTIONAL] Your "about-me" section looks very good well done 👏🏽 , there is only one small bug that still need to be fixed to match the Figma design, check the images below 🔽 

## Your Design
![2023-03-22_192416](https://user-images.githubusercontent.com/104319462/227015197-dd56e790-6e24-4a33-8582-2f47a017704f.png)

## Figma Design
![2023-03-22_192510](https://user-images.githubusercontent.com/104319462/227015214-51f612c4-693c-4856-a1db-91805b60f09f.png)

Reply:
@ouasamine "." have been removed and according to Figma design, it's accurate.